src/import/importd.c

```diff
@@ -94,6 +94,9 @@ struct Manager {
         int notify_fd;
 
         sd_event_source *notify_event_source;
+
+        bool setup_subvol;
+        bool setup_quota;
```

Comment:
Please add a brief comment that these are only for btrfs. Of course, "subvol" mostly implies btrfs, but `setup_quota` is slightly ambiguous.

Reply:
I'll just rename them such that it's clear.